pkgs/development/tools/misc/trv/default.nix

```diff
@@ -0,0 +1,36 @@
+{stdenv, fetchurl, ocaml, findlib, camlp4, core, async, async_unix, re2,
+  async_extra, sexplib, async_shell, core_extended, async_find, cohttp, uri, tzdata}:
+
+let
+  ocaml_version = (builtins.parseDrvName ocaml.name).version;
+  version = "0.1.3";
+in
+
+assert stdenv.lib.versionOlder "4.02" ocaml_version;
+
+stdenv.mkDerivation {
+  name = "trv-${version}";
+
+  src = fetchurl {
+   url = "https://github.com/afiniate/trv/archive/${version}.tar.gz";
+   sha256 = "1d6k4z3y8gdfi2xx8dvarayx7md2s102amp7nqs9qai62s6kjv7i";
+  };
+
+  buildInputs = [ ocaml findlib camlp4 ];
+  propagatedBuildInputs = [ core async async_unix
+                            async_extra sexplib async_shell core_extended
+                            async_find cohttp uri re2 ];
+
+  createFindlibDestdir = true;
+  dontStrip = true;
+
+  installPhase = "make SEMVER=${version} PREFIX=$out install";
```

Comment:
You can pass those variables using `installFlags` instead of overwriting the whole `installPhase`.